exercises/01-getting-started.md

```diff
@@ -1,63 +1,81 @@
 
-Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system.
+## Setup the destinations environment variable
 
+Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system. The mock server has no authentication to access it by default. Please set the authentication type `NoAuthentication` when you use the mock server.
 
 > ### How to find your mock server's URL
+>
 > Open the [SAP BTP Cockpit](https://account.hana.ondemand.com) and open the subaccount to which the mockserver was deployed. Open the space, where the mock server was deployed. You should find the URL to the mock server here.
 
 ## Start the application locally
-Open the project in your IDE. For the remainder of this course we will assume you are using Visual Studio Code. Please adapt accordingly if you are using a different IDE.
 
-To open the project go to *File > Open...* and select the folder you just cloned from GitHub.
+Open the project in your IDE. For the remainder of this tutorial, we will assume you are using Visual Studio Code. Please adapt accordingly if you are using a different IDE.
+
+To open the project go to `File` > `Open...` and select the folder you just cloned from GitHub.
 
-Let's check that everything works and run the application locally. First, open the command line with *Terminal > New Terminal*. Then, install the dependencies:
+Let's check that everything works and run the application locally. First, open the command line with `Terminal` > `New Terminal`. Then, install the dependencies:
 
 ```sh
 npm install
 ```
 
-Deploy a local database (locally we use sqlite):
+Deploy a local database with these steps: 
+
+1. create a local database (we use SQLite)
+2. drops existing tables and views, and re-creates them according to [your CDS model](../db/data-model.cds)
+3. deploy CSV files with initial data
+
+In your [`package.json`](../package.json), you have already installed `@sap/cds-dk` (SAP Cloud Application Programming Model (CAP)). SAP CAP provides a command to do the 3 steps at once. deploy a local database with the command:
```

Comment:
Don't abbreviate as 'CAP' as described [here](https://www.sapbrandtools.com/naming-center/#/requests/7815)

Reply:
removed the abbreviation and follow the naming-center